samplefigure.py

```diff
@@ -18,8 +77,23 @@
 plt.plot(x, ydata, 'b.', label='Experimental')
 plt.xlabel('Time / s')
 plt.ylabel('Height / m')
-plt.legend()
+plt.legend(fontsize=12)
 # This ensures the bounding box is correct. If we don't call tight_layout
 # sometimes elements get cut off on the edges.
 plt.tight_layout()
 plt.savefig('samplefigure.pdf')
+
+# Lets investigate the effect of noise on our Model "A" and it's derivative
+# For-loop structures are very useful for this type of sensitivty analysis
+# However, we can create a function to do most of this for us with ease
+# The functions are plotmodel() and set_axis_style() as above
+
+diffA = numpy.diff(modelA)
+diffB = numpy.diff(modelB)
+dx = x[1]-x[0]
+
+for index, value in enumerate([1.0, 10, 100]):
```

Comment:
We don't use index in this loop do we?

Reply:
Not any more no, it can be simplified down to
`for value in [1, 10, 100]:`